worker/include/handles/UdpSocketHandle.hpp

```diff
@@ -84,8 +84,10 @@ class UdpSocketHandle
 
 	/* Callbacks fired by UV events. */
 public:
+	// NOLINTNEXTLINE(readability-convert-member-functions-to-static)
 	void OnUvRecvAlloc(size_t suggestedSize, uv_buf_t* buf);
```

Comment:
Why is it ignored?

Reply:
Because it can be made `static` but then we would have this and `OnUvSend` static while `OnUvRecv` non static. I  added the ignore for simply for consistency.

Reply:
But that was my personal view. I can remove just `NOLINT` directives so the methods are converted to static and then we move them to the corresponding place in the file.

Reply:
I'm fine with your current code.